src/main/java/audio/harmonizer/Harmonizer.java

```diff
@@ -23,22 +23,22 @@ public InstrumentData harmonize() throws AppError {
         data.setInstrument(dataRaw.getInstrument());
 
         data.setPitch(normalizePitch(dataRaw.getPitch()));
```

Comment:
@Rex2002 Is there a reason why normalisePitch() still has a parameter when all other functions just get their arrays themselves?

Reply:
yes, I did that because it is not toplevel (defining harmonize() as toplevel), but beyond that there is none

Reply:
yeah, that actually makes sense :+1: ,
I currently merging, so I will include that there